testsuite/test_data/invalid_submission_license.yaml

```diff
@@ -0,0 +1,69 @@
+---
+
+additional_resources:
+  - location: "http:github.com/HEPData/hepdata"
+    description: "Full source code for creating this data"
+  - location: "http:github.com/HEPData/hepdata"
+    description: "Full source code for creating this data"
```

Comment:
All the test data has this invalid URL `http:github.com/HEPData/hepdata`.  Maybe correct it to `https://github.com/HEPData/hepdata`?

Reply:
Will do. I copied it from another test file - looks like there are lots of instances of this so I'll fix all of them (provided that's not the point of a test!).